CMakeLists.txt

```diff
@@ -930,7 +930,7 @@ if(LPCNET)
          set_tests_properties(test_freedv_reliable_text_awgn_1600 PROPERTIES PASS_REGULAR_EXPRESSION "1")
                             
     add_test(NAME test_freedv_reliable_text_fade_2020
-          COMMAND sh -c "cd ${CMAKE_CURRENT_SOURCE_DIR}/unittest; ./reliable_text_fade.sh 2020 -26 6 0 '${CMAKE_CURRENT_BINARY_DIR}/src'")
+          COMMAND sh -c "cd ${CMAKE_CURRENT_SOURCE_DIR}/unittest; ./reliable_text_fade.sh 2020 -26 4 0 '${CMAKE_CURRENT_BINARY_DIR}/src'")
 endif(LPCNET)
```

Comment:
Yes I've had a few fails on this test recently, with my inertnal (data mode) development.

Reply:
I also noticed that there aren't equivalent tests for 2020B/C, so that might be good to add too.